### DISCUSSION TITLE
Paper draft review

### DISCUSSION THREAD

**Discussion by taeil** [Berkeley-Data/irrigation_detection#5] [General]
## Abstract 
1. Same number of labeled images (doesn't sound like faire comparison)

## Intro 	
1.  Does not sound very strong: *This was a 10% improvement over the supervised baseline with the same number of labeled images. De- tailed sensitivities included an ablation study are also included herein*
2.  confirm the labeld source of data available (why not) 
3.  Is the problem general in US or World? 
4.  not discussing why irrigation detection is hard? (comparative to others or with satelite band, shouldn't it be easy simply checking the spectrum? 
	1.  manual: $7 label satelite images 
	2. why computer vision? 
5.  What other labelled satelite dataset other than BigEarthNet? 
	1.  
6.  major motivation: not sure strong enough 

## Related Work 
1. Doesn't seem to related to SimCLR. Why can we be generic CLS? 

## Data 
1. other band than agricultural bands? any explanation why? 
2. why BigEarthData set is not enough to demonstrate the effectiveness? 
3. CA data just for application? 

## Methods (need major improvement)

- bseline model: **- SOTA supervised model than our own? **
- 
- figure 3: no clear explanation of take away. is AUC right metrics for imbalanced data? F1-score sounds better 
-  SimCLR is not SOTA any more. .... 
-  Color distortion is counter-intuitive for agricultural band (shouldn't the model exploit the agricultural band) 
-  Maybe similar study as SimCLR for each component 
-  the pretrained SimCLR based model: this is not clear. 
-  baseline: confirmation bias 
-  figure 5: wrong legend
-  what classification task is with BigEarthNet data? same irrigation? 

6.2 (not so important proving the point)
- missing reference for t-SNE 
- figure 7... not so clear what is it...  maybe show images of agricultural band as well? 
- too small spaces for all 2048 images to be displayed (better representation) 
- validation mechanism is weak. 
- test pre-trained model from CA on BigEarthNet data without fine-tuning? Why finetune?
